### PR TITLE
Fix #42: Use HTML5 input types where possible

### LINKED ISSUE #42: Use HTML5 input types where possible
On email, phone number, website fields.

### PR DESCRIPTION
**Pull request by alastairs**
Turns out there weren't many places where this could be done, but it's useful for those fields that are of a particular type (e.g., email, URL, etc.). The fields are now auto-validated by the browser, and jQuery validation hooks into the validation API of the browser so error messages are displayed consistently.

### CODE REVIEW

**Comment by alastairs** [comment]
Note, I deliberately took this branch off `staging` to get @adrianbanks's latest changes and avoid conflicts. This should be fine provided.

File: DDDEastAnglia/Views/Profile/Edit.cshtml [hunk 1]
```diff
@@ -1,4 +1,5 @@
-﻿@model DDDEastAnglia.Models.UserProfile
+﻿@using System.ServiceModel.Channels
```
Comment (adrianbanks) [inline DDDEastAnglia/Views/Profile/Edit.cshtml]: What is this using for?

Comment (alastairs) [inline DDDEastAnglia/Views/Profile/Edit.cshtml]: No idea at all, didn't even see it get added. I'll pull it out tomorrow.

Comment (alastairs) [inline DDDEastAnglia/Views/Profile/Edit.cshtml]: Sorted.

**Pull request merged**